app/parse.py

```diff
@@ -1,4 +1,8 @@
 from dataclasses import dataclass
+from typing import List
```

Comment:
The import statement for `List` from `typing` is not used in the code. Consider removing it if it's unnecessary.